docs/_guides/codewind-crw-quick-guide.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: guide
+summary-title: "Codewind in CodeReady Workspaces"
+title: "Getting Started with Codewind in CodeReady Workspaces"
+categories: guides 
+guide_description: "Take advantage of Codewind's tools to help build high quality cloud native applications regardless of which IDE or language you use."
+permalink: codewind-crw-quick-guide.html
+duration: 10 minutes
+tags: Codewind, CodeReady Workspaces, Openshift, Kubernetes, containers, microservice
+--- 
+
+## Objectives
+* Install CodeReady Workspaces and Codewind
+* Develop a simple microservice that uses Eclipse Codewind on CodeReady Workspaces
+
+## Overview
+Use Eclipse Codewind to develop microservice applications from application stacks in an integrated developer environment (IDE). CodeReady Workspaces provides a containerized IDE for cloud native application development on an OpenShift cluster. 
+
+## Developing with CodeReady Workspaces 
+CodeReady Workspaces uses Kubernetes and containers to provide a preconfigured environment. Use CodeReady Workspaces to create, build, and test your code in OpenShift containers but feel like you are working on an IDE on your local computer.  
+
+### Prerequisite
+CodeReady Workspaces requires at least two 1Gi ReadWriteOnce (RWO) persistent volumes on the cluster to install and a 1Gi RWO volume for each created workspace.
+
+Each Codewind workspace also requires at least on 1Gi ReadWriteMany (RWX) persistent volume.
+
+### Installing CodeReady Workspaces
+1. Subscribe to the CodeReady Workspaces operator then install it.
+  * To manually install CodeReady Workspaces, apply the CodeReady Workspaces OperatorHub subscription. 
```

Comment:
Maybe we can take this sub point out? Let me also get you some screenshots for these steps, as I actually do think it will help now.